core/arch/arm/include/kernel/user_access_arch.h

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Amazon.com Inc. or its affiliates. All rights Reserved.
+ */
+
+#ifndef KERNEL_USER_ACCESS_ARCH_H
```

Comment:
nitpicking: prefer with `__` prefix to better match other recently updates .h header file in this dir.
```c
#ifndef __KERNEL_USER_ACCESS_ARCH_H
#define __KERNEL_USER_ACCESS_ARCH_H
(...)
#endif /*__KERNEL_USER_ACCESS_ARCH_H*/

Reply:
updated